docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -4,12 +4,113 @@
+
+[[i-deleted-my-agent]]
+=== I deleted my agent in the Fleet {ingest-manager} app so now what?
+
+Its ok, be cool: the data is still in Elasticsearch.  But to get the Agent back in the UI, you will need to stop, then re-enroll the Agent on the host and run it again.
+
+[[i-need-to-stop-agent]]
+=== I need to stop the Agent and all processes running on my host
+
+To stop the Agent and relating executables running on my hosts is as easy as stopping the Agent process.  If you installed it as a service on Windows you will need to manage that, as well.  After that, if needed, on Windows you can use Task Manager and shut down ‘Elastic Agent’ which will kill it and the sub-processes it created (beats).  On Linux / macOS you can kill the elastic-agent process running, look with a ps | grep elastic-agent call to get the id to kill
+
+[[i-rebooted-my-host]]
+=== I needed to reboot my host, I'm not sure if the Agent is still running.
+
+If you installed Windows Agent as a service with powershell, the Agent should still be running after a reboot of the host.  On macOS and Linux you will need to start the Agent again from the command line each time.  The ability to enhance support for those operating systems is known, as a temporary fix you could add that start command to a start up profile.
+
+[[what-is-the-endpoint-package]]
+=== The {ingest-manager} app shows an Integration called 'Endpoint' and I want to use it.
+
+We all want to use it!  But we can't in 7.8 - It is not yet even 'Alpha' state, as the {ingest-manager} app is.  For some context, Endpoint is the integration that will allow the Elastic Security app to have a dedicated ‘Endpoint’ executable running like a beat and will be managed by the Agent as other beats are. It cannot yet be used in 7.8 so it is best left alone.  It won't hurt anything sitting there, promise.
+
+[[ingest-manager-not-in-on-prem-kibana]]
+=== The {ingest-manager} app is not listed in my Kibana on-premises navigation list.
+
+In 7.8, the {ingest-manager} app is in Alpha state and as such is only shown when specifically enabled in the start up config.  To do this you must modify both the Elasticsearch and Kibana configs as well as adhering to all 'security enabled' requirements, like modifying the default elastic user password, see below for details:
```

Comment:
Might be a good chance to get in a plug for cloud here. :-)

Reply:
Yes, lets go for beta. :)

Reply:
Cloud is plugged above, the first thing I listed.  :) If we want to reference that or a more direct plug I'm open, I don't know what is appropriate or feasible for cross-doc or web site linking etc.